documentation/api/commands.markdown

```diff
@@ -90,9 +90,23 @@ payload of this command.
 
 ### "deactivate node", version 2
 
-The payload is expected to be the certname of a node, as a raw JSON
-string, which will be deactivated effective as of the time the command
-is *processed*.
+The payload is a JSON map, formatted as follows:
+
+     {
+      "certname": <string>,
+      "producer_timestamp": <datetime>
+     }
```

Comment:
Should we fork out a new document to contain these details like we have for the other wire formats? I think up until now there has been minimal complexity so we haven't needed it.